source/index.html

```diff
@@ -153,30 +158,32 @@ <h2 class="example__title">
+        <a class="main-footer__link" href="#">
+          <svg class="main-footer__youtube-icon" aria-hidden="true" focusable="false" width="28" height="22" fill="#666666" xmlns="http://www.w3.org/2000/svg"><path d="m17.85 10.8-5.24-3.5a.65.65 0 0 0-1.01.55v6.98a.65.65 0 0 0 1.01.54l5.24-3.49a.65.65 0 0 0 0-1.09Zm-4.94 2.8V9.08l3.4 2.27-3.4 2.27Zm12.48-8.56a2.4 2.4 0 0 0-1.45-1.63C20.22 1.98 14.25 2 14 2s-6.22-.02-9.94 1.4a2.4 2.4 0 0 0-1.45 1.64 25.32 25.32 0 0 0-.61 6.3c0 3.26.33 5.22.61 6.3a2.4 2.4 0 0 0 1.45 1.63c3.72 1.43 9.69 1.4 9.94 1.4h.08c.75 0 6.31-.04 9.86-1.4a2.4 2.4 0 0 0 1.45-1.64c.28-1.07.61-3.03.61-6.3 0-3.25-.33-5.22-.61-6.29Zm-1.27 12.27a1.1 1.1 0 0 1-.65.74c-3.5 1.34-9.4 1.32-9.47 1.32-.07 0-5.98.02-9.47-1.32a1.09 1.09 0 0 1-.65-.74c-.27-1-.57-2.85-.57-5.97 0-3.12.3-4.97.57-5.97a1.1 1.1 0 0 1 .65-.74C8.03 3.29 13.93 3.3 14 3.3c.07 0 5.98-.02 9.47 1.32a1.1 1.1 0 0 1 .65.74c.27 1 .57 2.85.57 5.97 0 3.12-.3 4.96-.57 5.97Z"/></svg>
           <span class="visually-hidden">
             Ютуб
           </span>
         </a>
-        <a class="main-footer__link telegram-link" href="#">
-          <svg class="vk-link__icon" aria-hidden="true" focusable="false" width="28" height="22" fill="#666666" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M23.46 3.15c.18.15.27.38.23.61l-2.63 15.76a.66.66 0 0 1-1.01.44l-7.88-5.25a.66.66 0 0 1-.15-.96l2.61-3.26-4.36 2.9a.66.66 0 0 1-.65.05L4.36 10.8a.66.66 0 0 1 .08-1.2l18.38-6.57a.66.66 0 0 1 .64.11ZM6.33 10.32l3.53 1.77 7.56-5.04a.66.66 0 0 1 .88.96l-4.8 6 6.43 4.3 2.28-13.66-15.88 5.67Z"/></svg>
+        <a class="main-footer__link" href="#">
+          <svg class="main-footer__telegram-icon" aria-hidden="true" focusable="false" width="28" height="22" fill="#666666" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M23.46 3.15c.18.15.27.38.23.61l-2.63 15.76a.66.66 0 0 1-1.01.44l-7.88-5.25a.66.66 0 0 1-.15-.96l2.61-3.26-4.36 2.9a.66.66 0 0 1-.65.05L4.36 10.8a.66.66 0 0 1 .08-1.2l18.38-6.57a.66.66 0 0 1 .64.11ZM6.33 10.32l3.53 1.77 7.56-5.04a.66.66 0 0 1 .88.96l-4.8 6 6.43 4.3 2.28-13.66-15.88 5.67Z"/></svg>
           <span class="visually-hidden">
             Телеграм
           </span>
         </a>
       </div>
-      <a class="main-footer__html-logo html-link" href="https://htmlacademy.ru/">
-          <p class="html-link__text">
+      <a class="main-footer__html-logo" href="https://htmlacademy.ru/">
+          <p class="main-footer__html-text">
```

Comment:
Более подходящими будут следующие имена элементов:
```suggestion
      <a class="main-footer__company-logo" href="https://htmlacademy.ru/">
          <p class="main-footer__company-text">
```